cmd/bblfshd/main.go

```diff
@@ -5,6 +5,8 @@ import (
 	"flag"
 	"fmt"
 	"net"
+	"net/http"
+	_ "net/http/pprof"
```

Comment:
I suggest grouping the blank (side-effecting) imports separately, below the others.